generators/app/templates/src/app/screens/OnBoarding/index.js

```diff
@@ -0,0 +1,20 @@
+import React, { memo } from 'react';
+import PropTypes from 'prop-types';
+import Routes from '@constants/routes';
+
+import './i18n';
+import Swiper from './components/Swiper';
+
+function OnBoardingContainer({ navigation: { navigate } }) {
+  const skipOnBoarding = () => navigate(Routes.App);
```

Comment:
Use `useCallback`

Reply:
I don't think it's a good idea to use `useCallback` here. This will render only once, it's the root of the onBoarding

Reply:
Use `useCallback` here will take unnecessary space